config/pulumi/Pulumi.stackname.yaml.example

```diff
@@ -13,6 +13,20 @@
 # NOTE: Currently, many of the stacks stood up by this process have sanity checks
 #       that will fill in default values if values are not found in this file.
 #
+# IMPORTANT NOTE ON NAMESPACE NAMES!
+#
+# You CANNOT name your namespace with the same name as a Pulumi provider. As an
+# example, you cannot call your AWS namespace "aws" or your Digital Ocean
+# namespace "digitalocean". If you do so you will get undefined and bizarre errors
+# back from Pulumi at runtime.
```

Comment:
**Suggestion**:
Can we list out all the disallowed namespaces for the providers we support at the moment?

Reply:
Added the ones that I know of currently and added some more text around the problem.